src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im2.contexts.onboarding.new-to-status.view
+  (:require [quo2.components.markdown.text :as text]
```

Comment:
@ulisesmac please use from quo2.core for components. e.g 
require [quo2.core :as quo]
...
 (quo/text {...} "")

Reply:
Will do it! thanks!